src/main/java/ch/njol/skript/expressions/ExprTimeDifference.java

```diff
@@ -53,7 +69,7 @@ public Timespan convert(Date date) {
 		 * A value of 0 indicates that the new date is the SAME as the current date.
 		 * A value greater than 0 indicates that the new date is AFTER the current date.
 		 */
-		if (date.compareTo(now) < 1)
+		if (sinceDate ? (date.compareTo(now) < 1) : (date.compareTo(now) > -1))
 			return date.difference(now);
 		return null;
```

Comment:
This check is no longer needed since now this expression supports future dates
```suggestion
		return date.difference(now);
```

Reply:
The check is there so that `time since (10 seconds from now)` doesn't return a value and keeps the old functionality, if you think it's better without then that's fine

Reply:
I'm in favor of keeping the checks

it prevents behavior from weird things like `time since 10 seconds later` or `time until 10 seconds ago`